mne/preprocessing/tests/test_ica.py

```diff
@@ -608,4 +608,27 @@ def test_fit_params():
     ICA(fit_params=fit_params)  # test no side effects
     assert_equal(fit_params, {})
 
+
+@requires_sklearn
+def test_bad_channels():
```

Comment:
You can fix the random_state, see how it's done in other ICA tests. I'm not
sure what exactly you'd want to test, maybe that it's actually including
the EOG channel or something. Maybe the other existing ICA tests will give
you some ideas.